api/capi/capi-ml-training.pc.in

```diff
@@ -4,7 +4,7 @@ exec_prefix=@PREFIX@
 libdir=@LIB_INSTALL_DIR@
 includedir=@INCLUDE_INSTALL_DIR@
 
-Name: tizen-api-nntrainer
```

Comment:
Need to check if it is ok to change the name. maintaining compatibility.

Reply:
IMO this should be fine, as the name of the pc file is used searching for a plugin.

Reply:
AFAIK, this has possibility to break, rpm allows `pkgconfig(pkgname)` which might be already included in some tizen middleware, I guess we could check during SR?

Reply:
Checked offline it is more likely to be safe change.

Reply:
This wont be a problem. `pkgconfig(pkgname)` -> pkgname is the name of the pc file.